src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double area() {

```

Comment:
The method name 'area' does not follow the java naming conventions. Method names should be verbs and start with a lowercase letter. In addition, it's better to make the method name more descriptive. Consider renaming it to 'calculateArea'.